lib/plenario2/actions/data_set_actions.ex

```diff
@@ -81,6 +81,7 @@ defmodule Plenario2.Actions.DataSetActions do
 
       {:error, error} ->
         Logger.error("error creating table: #{error.postgres.message}", table_name: table_name)
+        Logger.error(sql)
```

Comment:
When I was fixing the bug in the `add_constraint` function, this helped me figure out what was going wrong with the template.